rbs/CommentsAssociator.h

```diff
@@ -24,11 +25,13 @@ class CommentsAssociator {
     static const std::string_view ANNOTATION_PREFIX;
     static const std::string_view MULTILINE_RBS_PREFIX;
     static const std::string_view BIND_PREFIX;
+    static const std::regex TYPE_ALIAS_PATTERN;
 
     core::MutableContext ctx;
     std::vector<core::LocOffsets> commentLocations;
     std::map<int, CommentNode> commentByLine;
     std::map<parser::Node *, std::vector<CommentNode>> commentsByNode;
+    std::vector<parser::Node *> nesting;
```

Comment:
```suggestion
    std::vector<const parser::Node *> nesting;
```

I don't think you plan on mutating the tree indirectly via the nesting scope, so we should probably keep this `const`.